schema.sql

```diff
@@ -11,3 +11,18 @@ CREATE TABLE species(id SERIAL PRIMARY KEY, name VARCHAR(255));
 ALTER TABLE animals DROP COLUMN species;
 ALTER TABLE animals ADD COLUMN species_id INTEGER REFERENCES species(id);
 ALTER TABLE animals ADD COLUMN owner_id INTEGER REFERENCES owners(id);
+
+CREATE TABLE vets(id SERIAL PRIMARY KEY, name VARCHAR(255), age INTEGER, date_of_graduation DATE);
+
+CREATE TABLE visits(animal_id INTEGER,
+  vet_id INTEGER, 
+  PRIMARY KEY(animal_id, vet_id),
+  FOREIGN KEY (animal_id) REFERENCES animals (id)
+  FOREIGN KEY (vet_id) REFERENCES vets (id));
```

Comment:
- [ ] The join table Visits should also keep track of the date of the visit, Kindly add date_of_visit to the table.